.github/workflows/test.yml

```diff
@@ -24,14 +24,17 @@ jobs:
           file: coverage.out
           fail_ci_if_error: false
   e2e-minikube:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
I thought 20.04 is the latest. So is this change needed? Or you want to pin a version for a reason?

Reply:
I was tempted to add a review comment note for this change as it's technically a no-op (latest is the 20.04 LTS release) but I'm typically a fan of using explicit versions rather than relying on potentially moving latest tags. I don't have a strong opinion either way so I'm fine to revert.

Reply:
I'm fine with it given it is a LTS version. However, we do need to manually update this if a newer version requires in the future. I don't think it is gonna be an issue.